src/modals/InfoModal.tsx

```diff
@@ -1,93 +1,143 @@
-          </IonCardContent>
-        </IonCard>
-      </div>
-      <div id="small-rectangle"></div>
-      <div id="rectangle">
-        <IonCard>
-          <IonCardHeader class="info">{t("Frequent symptoms")}</IonCardHeader>
-          <IonCardContent style={{ textAlign: "justify" }}>
-            <SymptomsList symptoms={phase.symptoms} />
-          </IonCardContent>
-        </IonCard>
-      </div>
-    </>
-  );
-};
-
-interface PropsSymptoms {
-  symptoms: string[];
-}
-
-const SymptomsList = (props: PropsSymptoms) => {
-  const list = props.symptoms.map((item, idx) => <p key={idx}>{item}</p>);
-
-  return <>{list}</>;
-};
-
 interface PropsInfoModal {
   isOpen: boolean;
   setIsOpen: (newIsOpen: boolean) => void;
 }
 
 const InfoModal = (props: PropsInfoModal) => {
+  const { t } = useTranslation();
+  const cycles = useContext(CyclesContext).cycles;
+
+  const lengthOfCycle = getAverageLengthOfCycle(cycles);
+  const currentDay = getDayOfCycle(cycles);
+  const ovulationStatus = getOvulationStatus(cycles);
+  const pregnancyChance = getPregnancyChance(cycles);
+
+  const phase = getPhase(cycles);
+
   return (
     <IonModal
       id="info-modal"
       backdropDismiss={false}
       isOpen={props.isOpen}
     >
-      <div id="small-rectangle"></div>
       <IonContent
         className="ion-padding"
-        color="basic"
+        color="background"
       >
-        <Phase />
-        <div id="small-rectangle"></div>
-        <IonButton
-          class="ok-modal"
-          color="dark-basic"
-          onClick={() => props.setIsOpen(false)}
+        <p
+          style={{
+            fontWeight: "bold",
+            fontSize: "25px",
+            color: "var(--ion-color-dark-basic)",
+            marginBottom: "24px",
+          }}
+        >
+          {t("Days_interval", {
+            postProcess: "interval",
+            count: 1, // NOTE: to indicate which day is in the account, you need to write the day as if in the singular
+          })}{" "}
+          {currentDay}/{lengthOfCycle}
+        </p>
+        <ul>
+          <li
+            style={{
+              fontSize: "16px",
+              color: "var(--ion-color-dark)",
+              marginBottom: "20px",
+            }}
+          >
+            <span
+              style={{
+                color: "var(--ion-color-dark-basic)",
+                fontWeight: "bold",
+              }}
+            >
+              {phase.title}
+            </span>
+            <span> {t("is current phase of cycle")}</span>
+          </li>
+          <li
+            style={{
+              fontSize: "16px",
+              color: "var(--ion-color-dark)",
+              marginBottom: "20px",
+            }}
+          >
+            <span>{t("Ovulation")}</span>
+            <span
+              style={{
+                color: "var(--ion-color-dark-basic)",
+                fontWeight: "bold",
+              }}
+            >
+              {" "}
+              {ovulationStatus}
```

Comment:
{`${ovulationStatus`}

If you need one empty line than use be tag, some component from ionic or setup it with css style

Reply:
Fixed